.github/workflows/cron_weekly.yml

```
@@ -26,9 +26,9 @@ jobs:
           - {os: ubuntu-22.04, compiler: gcc-latest-cpp20-release-open_issue}
     steps:
       - name: Standard IV-project testing
-        uses: iv-project/IVaction@v11.1
+        uses: iv-project/IVaction@v12.1
         with:
           compiler: ${{ matrix.compiler }}
           threads: 2
           cmake_flags: ${{ matrix.cmake_flags }}
```

actionlint (1.7.12)

[error] property "cmake_flags" is not defined in object type {compiler: string; os: string}

(expression)